.github/workflows/ci-build.yaml

```diff
@@ -154,6 +154,33 @@ jobs:
         env:
           CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
 
+  build-binaries:
+    name: Build Binaries
+    needs: [ changed-files ]
+    if: ${{ needs.changed-files.outputs.e2e-tests == 'true' }}
+    runs-on: ubuntu-24.04
+    timeout-minutes: 10
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - target: controller
+          - target: cli
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
+        with:
+          go-version: "1.23"
+          cache: true
+      - name: Build
+        run: make ${{matrix.target}}
```

Comment:
Is `STATIC_FILES=false` being left out on purpose?
Having it should make the build faster

Reply:
It's actually set to false here, so no need to explicitly set it: https://github.com/argoproj/argo-workflows/blob/abd14b7801647984859fd33e738a060199b40d9a/Makefile#L44-L49